### ISSUE TITLE
[A11y]: Labels should be provided for input boxes and select boxes under members

### ISSUE THREAD

**Issue by Goodyear2017** [NuGet/NuGetGallery#9482]
### Impact

Other

### Describe the bug

Labels for inputs and selects are hidden.

### Repro Steps

Steps to reproduce the issue:
1. After signing in from top right of any page, select "Manage Organizations" from dropdown.
2. click "Members."
3. All the labels are invisible

### Expected Behavior

All the labels should be visible.

**Comment by agr** [comment]
I don't quite understand what the issue is, the user impact in the original issue states:

> Screen reader users will never know about the purpose of the controls as label is not provided for combobox and edit box present under members.

But the controls mentioned have `aria-label` attribute set explaining their purpose. Also, the linked accessibility requirements state that "label or *instructions*" are to be provided. We do provide instructions already (unless screen reader not reading it). Should we clarify the issue with the accessibility team?
